### PR TITLE
617 switch to vue

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@Lenseg @kazurus

File: src/components/screen/Oauth.vue [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                  {{ account }}
+                </li>
+              </ul>
+            </div>
+            <div>
+              <a
+                class="button is-danger"
+                @click="onLogout"
+              >
+                Logout
```
Comment (kazurus) [inline src/components/screen/Oauth.vue]: Close

**Comment by kazurus** [comment]
@dudiq :recycle:

**Comment by kazurus** [comment]
@dudiq :recycle:

**Pull request merged**